packages/secrets/src/keyring/EXTENDERS.md

```diff
@@ -63,6 +63,18 @@ await keyring.deletePassword("TestService", "AccountA");
 
 ## Webpacking/bundling alongside your project
 
+**Note for Webpack users:** If you do **_not_** want to use the Secrets SDK, but have Imperative modules imported that reference it (such as `ConvertV1Profiles` or `DefaultCredentialManager`), you must define the `@zowe/secrets-for-zowe-sdk` package in the `externals` section of your Webpack config:
+
+```json
+"externals": {
+    "@zowe/secrets-for-zowe-sdk": "commonjs @zowe/secrets-for-zowe-sdk"
+}
+```
+
+This will prevent Webpack from trying to dynamically resolve the Secrets SDK during compile time. This does not affect the majority of developers, as Webpack will omit any unused Imperative modules during the bundling phase.
+
+---
```

Comment:
Should we use GitHub note syntax, or do we want to avoid that since not all MD previewers support it?
https://github.com/orgs/community/discussions/16925

Reply:
I like the idea, but we can probably pass on this for now just so it looks the same in every MD previewer. If we use it elsewhere in `.md` files, then I'll update it to be consistent 😋